DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/Index.cshtml

```diff
@@ -45,24 +45,78 @@
       </div>
     </div>
 
+
+    <div class="nhsuk-grid-row">
+      <form method="get" role="search">
+        <input type="hidden" name="sortBy" value="@Model.SortBy" />
+        <input type="hidden" name="sortDirection" value="@Model.SortDirection" />
+        <div id="search">
+          <label class="nhsuk-label" for="search-field">Search:</label>
+          <input class="search-box" id="search-field" name="searchString" type="search" placeholder="Search" autocomplete="off" value="@Model.SearchString">
+          <button class="nhsuk-search__submit non-js-only" type="submit" asp-action="Index" asp-route-page="1">
+            <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+              <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
+            </svg>
+            <span class="nhsuk-u-visually-hidden">Search</span>
+          </button>
+        </div>
+      </form>
+    </div>
+
     <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-full">
-        @if (!Model.Delegates.Any()) {
-          <p class="nhsuk-u-margin-top-4" role="alert">
-            <b>No delegates found</b>
-          </p>
-        } else {
-          <div id="searchable-elements">
-            @foreach (var delegateModel in Model.Delegates) {
-              <partial name="_SearchableDelegateCard" model="delegateModel" />
-            }
+      <form method="get" role="search">
+        <input type="hidden" name="searchString" value="@Model.SearchString" />
+        <div class="nhsuk-grid-column-full nhsuk-form-group Xsort-row">
+          <label class="nhsuk-label" for="select-sort-by">
+            Sort by:
+          </label>
+          <select class="nhsuk-select nhsuk-search__input nhsuk-search__input--withdropdown sort-dropdown" id="select-sort-by" asp-for="SortBy" asp-items="Model.SortBySelectListItems" aria-label="Sort by"></select>
+          <select class="nhsuk-select nhsuk-search__input nhsuk-search__input--withdropdown sort-dropdown" id="select-sort-direction" asp-for="SortDirection" aria-label="Sort direction">
+            <option value="Ascending">Ascending</option>
+            <option value="Descending">Descending</option>
+          </select>
+          <div class="non-js-only">
+            <button class="nhsuk-button nhsuk-button--secondary button-small" type="submit">
+              Sort
+            </button>
           </div>
-        }
-      </div>
+        </div>
+      </form>

```

Comment:
I never made a new partial for Sort, but we should create one in the same place as the Pagination partial (this looks like it was just copied from the _CoursePageSort partial that it would be a copy of anyway).